### PR TITLE
update subgraph-channels

### PR DESCRIPTION
**Pull request by Eliaxie**
- Added `Account.isActive` flag and `Account.openChannelsCount` to Subgraph-channels
- Fixed Subgraph-channels compile errors (maybe due to a new TheGraph version)
- deployed [subgraph](https://thegraph.com/hosted-service/subgraph/eliaxie/hopr-channels)

(created another PR for the rebrand branch at https://github.com/hoprnet/hopr-community/pull/156)

### CODE REVIEW

**Comment by nionis** [comment]
hey @Eliaxie, could you move the projects into `project` folder and also resolve conflicts?

**Review by nionis — COMMENTED** [review]
left 1 comment, could you also link the resulting subgraph in `subgraph-channels/README.md`

File: subgraph-channels/src/mapping.ts [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+
     if (multiaddr.indexOf(event.params.multiaddr) == -1) {
         multiaddr.push(event.params.multiaddr)
     }
     account.multiaddr = multiaddr
     account.publicKey = event.params.publicKey;
     account.hasAnnounced = true;
     account.save()
-}8
+} 8
```
[the remark this answers is not in the document]
Comment (Eliaxie) [reply subgraph-channels/src/mapping.ts:20]: Idk hahaha
It surprised me too, maybe it's a signature so I left it in there, we need to ask @QYuQianchen

**Comment by Eliaxie** [comment]
Added resulting subgraph, keep in mind that as per [TheGraph Archive Policy](https://thegraph.com/docs/en/hosted-service/deploy-subgraph-hosted/#subgraph-archive-policy) the link could expire

File: subgraph-channels/src/mapping.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (nionis) [reply subgraph-channels/src/mapping.ts:20]: lets remove it, we can then merge this PR

**Review by nionis — APPROVED** [review]

**Pull request merged**